core/cafe_api_test.go

```diff
@@ -68,8 +68,21 @@ func TestCafeApi_Setup(t *testing.T) {
 	// wait for cafe to be online
 	<-node2.OnlineCh()
 
-	// register cafe
-	if _, err := node1.RegisterCafe("http://127.0.0.1:5000"); err != nil {
+	// create token on cafe
+	token, err := node2.CreateCafeToken()
+	if err != nil {
+		t.Error(fmt.Errorf("error creating cafe token: %s", err))
+		return
+	}
+
+	ok, err := node2.CompareCafeDevToken(token)
```

Comment:
This is just an additional check in the test, not required to just get the test working.